packaging/rpm/microshift.spec

```diff
@@ -336,12 +357,19 @@ systemctl enable --now --quiet openvswitch || true
 %config(noreplace) %{_sysconfdir}/microshift/lvmd.yaml.default
 %config(noreplace) %{_sysconfdir}/microshift/ovn.yaml.default
 
+%dir %{_datadir}/microshift
+%dir %{_datadir}/microshift/spec
```

Comment:
What goes into the spec dir?

Reply:
Nothing from this PR. I was making sure that all directories are declared.
Otherwise, they're not cleaned on uninstall.

Reply:
If you're asking a general question.
```
$ ls -l /usr/share/microshift/spec/
total 8
-rw-r--r--. 1 root root 5002 Nov 21 14:41 config-openapi-spec.json
```